src/main/kotlin/io/reactivex/rxkotlin/flowable.kt

```diff
@@ -187,3 +187,21 @@ fun <A: Any, B: Any> Flowable<Pair<A, B>>.toMultimap() = toMultimap({it.first},{
 @BackpressureSupport(BackpressureKind.FULL)
 @SchedulerSupport(SchedulerSupport.NONE)
 fun <T : Any> Iterable<Publisher<T>>.concatAll() = Flowable.concat(this)
+
+/**
+ * Sort flowable ascending with specified comparable key
+ */
+@CheckReturnValue
+@SchedulerSupport(SchedulerSupport.NONE)
+inline fun <T : Any, K : Comparable<K>> Flowable<T>.sortedBy(crossinline keySelector: (T) -> K): Flowable<T> {
+    return sorted { left, right -> keySelector(left).compareTo(keySelector(right)) }
+}
+
+/**
+ * Sort flowable descending with specified comparable key
+ */
+@CheckReturnValue
+@SchedulerSupport(SchedulerSupport.NONE)
+inline fun <T : Any, K : Comparable<K>> Flowable<T>.sortedByDescending(crossinline keySelector: (T) -> K): Flowable<T> {
```

Comment:
Maybe it'll be better to add extra param `ascending: Boolean = true` to sortedBy, to combine those two functions in one?

Reply:
No